Samples/android/Kotlin/Android-SDK-Demo-App/app-v7-java/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.AndroidSDKDemoApp" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
```

Comment:
The start page is rendered in dark mode is almost impossible to make anything out of it. I suggest changing `parent` to `Theme.MaterialComponents.Light.NoActionBar` so it stays consistent with the Kotlin version.

Reply:
@DavidKimVungle I'm not clear about this. The themes.xml is used for night mode, so it should use Dark theme. I cross-checked kotlin-->values-night/themes.xml, it is also `parent="Theme.MaterialComponents.DayNight.DarkActionBar"`